src/Helpers/JobTypes.cs

```diff
@@ -36,12 +54,57 @@ public class RetriableJob
             .StartNow()
             .WithSimpleSchedule(opts =>
                 opts
-                    .WithIntervalInSeconds(intervalInSeconds)
-                    .WithRepeatCount(retryTimes))
+                    .WithIntervalInMinutes(intervalListInMinutes[0])
+                    .WithRepeatCount(intervalListInMinutes.Length))
             .Build();
 
         return new JobAndTrigger(job, trigger);
     }
+
+    public static int[]? ParseRetryListFromEnvironmenVariable(string variable)
+    {
+        var retryListAsString = Environment.GetEnvironmentVariable(variable);
+        return retryListAsString?
+            .Split(",")
+            .Select<string, int>(s => int.Parse(s))
+            .ToArray();
+    }
+
+    /// <summary>
+    /// Call this function inside your Job's Execute method to set up retries and execute your action.
+    /// </summary>
+    /// <param name="context">The execution context of the job</param>
+    /// <param name="f">The action you want to perform inside the job</param>
+    public static async Task Execute(IJobExecutionContext context, Func<Task> f)
```

Comment:
f is a single char for a variable name, I would recommend writing what it does in a c#-ish way, like `function` or `callback`

Reply:
I agree, I'll change it